packages/esm-generic-patient-widgets-app/src/obs-table/obs-table.component.tsx

```diff
@@ -20,6 +21,20 @@ interface ObsTableProps {
   patientUuid: string;
 }
 
+interface Row {
+  id: string;
+  date: string;
```

Comment:
```suggestion
  date: string;
  rawDate?: string;
```

Might be useful to explicitly declare this optional property in the interface because we're using it in the sort function.